connector-packager/connector_packager/jar_jdk_signer.py

```diff
@@ -49,12 +53,12 @@ def get_user_pwd(alias):
         return str.encode(ks_pwd + "\n"), str.encode(alias_pwd + "\n")
 
 
-def jdk_sign_jar(input_dir, taco_name, alias, keystore):
+def jdk_sign_jar(input_dir: Path, taco_name: str, alias: str, keystore: str) -> bool:
```

Comment:
I would expect the hinting to be the same as validate_signing_input (alias and keystore being _Optional[str]_). Is the difference intentional?

Reply:
Let's dbl check with @jsun28 but I _think_ they're all required in the function. Will change if not :)

Reply:
All parameters in this function are required.